scripts/init.d-xapissl

```diff
@@ -128,6 +127,12 @@ EOF
             # If we had v5.06 or newer we would need to allow SSLv3 with
             # writec 'options -NO_SSLv3'
             # but all earlier versions choke on that option.
+
+            # Only give option 'LEGACY_CLIENT_ACCEPT' when ssl-legacy == true
+            # and CCM installed
+            if ldd ${STUNNEL} | grep -q "citrix\/libssl"; then
```

Comment:
I don't think we need the backslash here: `grep -q 'citrix/libssl'` would be better. It would also work with double quotes (at we have now) because '/` is not a character that needs escaping on the shell or for grep.